src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -63,12 +63,12 @@ public Parameter[] GetOrderedParametersByRequired()
             return OrderParametersByRequired(_parameters.Values);
         }
 
-        public static IEnumerable<InputParameter> GetParametersFromOperations(IEnumerable<InputOperation> operations) =>
-            operations
-                .SelectMany(op => op.Parameters)
-                .Where(p => p.Kind == InputOperationParameterKind.Client)
-                .Distinct()
-                .ToList();
+        public static IEnumerable<InputParameter> GetParametersFromClient(InputClient client)
+        {
+            return (client.Parameters ?? new List<InputParameter>())
```

Comment:
I think we should use `Array.Empty<>()` here.
And also we should make `Parameters` non-nullable (in the deserialization) on the input types so that here you do not have to do this.